src/coreclr/inc/corhdr.h

```diff
@@ -1141,7 +1141,6 @@ typedef enum CorExceptionFlag                       // definitions for the Flags
     COR_ILEXCEPTION_CLAUSE_FILTER  = 0x0001,        // If this bit is on, then this EH entry is for a filter
     COR_ILEXCEPTION_CLAUSE_FINALLY = 0x0002,        // This clause is a finally clause
     COR_ILEXCEPTION_CLAUSE_FAULT = 0x0004,          // Fault clause (finally that is called on exception only)
-    COR_ILEXCEPTION_CLAUSE_DUPLICATED = 0x0008,     // duplicated clause. This clause was duplicated to a funclet which was pulled out of line
```

Comment:
Nit: You can delete `COR_ILEXCEPTION_CLAUSE_OFFSETLEN` and `COR_ILEXCEPTION_CLAUSE_DEPRECATED` above while you are on it.